Sources/MapboxNavigation/Style.swift

```diff
@@ -49,6 +49,14 @@ open class Style: NSObject {
         UITraitCollection(userInterfaceIdiom: .pad),
     ])
     
+    class var defaultBorderWidth: CGFloat {
+        1 / UIScreen.main.scale
+    }
```

Comment:
The original purpose of the shadow was to ensure contrast with a map background that could be the exact same color. A border can accomplish the same contrast, though we should do some field-testing to verify that the border is thick and dark enough to make the button discernible in a sunny, washed-out lighting condition.